0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -186,11 +186,13 @@ Before anything else, when a client connects to a server, it sends a special
 `Message` request. Its primary `protocolData` entry MUST have name `'auth'`,
 content type `MIME_APPLICATION_OCTET_STREAM`,
 and empty data, and among the secondary entries, there MUST be a UTF-8
-`'auth_token'` entry, and a UTF-8 `'auth_username'` entry. The further secondary
+`'auth_token'` entry, and there MAY be a UTF-8 `'auth_username'` entry. The further secondary
```

Comment:
These two statements contradict each other: 
> [...] there MAY be a UTF-8 `'auth_username'` entry.

and later 

>[...] `'auth_username'` data can be set to the empty string, but they cannot be omitted.

Reply:
ah, good catch! fixed.